index.html

```diff
@@ -0,0 +1,170 @@
+            </div>
+            <p>A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required.</p>
+            <ul class="diff_classes">
+                <li><h4>html</h4></li>
+                <li><h4>css</h4></li>
+                <li><h4>javaScript</h4></li>
+            </ul>
+            <h3>See Project</h3>
+        </div>
+        <div class="grid-item item2">
+            <img src='./images/portfolio2.svg' alt="portfolio image">
+            <h1>Tonic</h1>
+            <div>
+                <h4>CANOPY</h4>
+                <ul>
+                    <li><img src='./images/Counter.svg' alt="Counter image"></li>
+                    <li>&nbsp;&nbsp;Back End Dev</li>
+                    <li>&nbsp;&nbsp;<img src='./images/Counter.svg' alt="Counter image"></li>
+                    <li>&nbsp;&nbsp;2015</li>
+                </ul>
+            </div>
+            <p>A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required.</p>
+            <ul class="diff_classes">
+                <li><h4>html</h4></li>
+                <li><h4>css</h4></li>
+                <li><h4>javaScript</h4></li>
+            </ul>
+            <h3>See Project</h3>
+        </div>
+        <div class="grid-item item3">
+            <img src='./images/portfolio3.svg' alt="portfolio image">
+            <h1>Tonic</h1>
+            <div>
+                <h4>CANOPY</h4>
+                <ul>
+                    <li><img src='./images/Counter.svg' alt="Counter image"></li>
+                    <li>&nbsp;&nbsp;Back End Dev</li>
+                    <li>&nbsp;&nbsp;<img src='./images/Counter.svg' alt="Counter image"></li>
+                    <li>&nbsp;&nbsp;2015</li>
+                </ul>
+            </div>
+            <p>A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required.</p>
+            <ul class="diff_classes">
+                <li><h4>html</h4></li>
+                <li><h4>css</h4></li>
+                <li><h4>javaScript</h4></li>
+            </ul>
+            <h3>See Project</h3>
+        </div>  
+        <div class="grid-item item4">
+            <img src='./images/portfolio4.svg' alt="portfolio image">
+            <h1>Tonic</h1>
+            <div>
+                <h4>CANOPY</h4>
+                <ul>
+                    <li><img src='./images/Counter.svg' alt="Counter image"></li>
+                    <li>&nbsp;&nbsp;Back End Dev</li>
+                    <li>&nbsp;&nbsp;<img src='./images/Counter.svg' alt="Counter image"></li>
+                    <li>&nbsp;&nbsp;2015</li>
+                </ul>
+            </div>
+            <p>A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required.</p>
+            <ul class="diff_classes">
+                <li><h4>html</h4></li>
+                <li><h4>css</h4></li>
+                <li><h4>javaScript</h4></li>
+            </ul>
+            <h3>See Project</h3>
```

Comment:
- Should be a `` button`` or ``a`` tag.

Reply:
I have used the hover effect on the buttons as per the suggestion.